docs/man_pages/screenbuilder/add-field.md

```diff
@@ -0,0 +1,22 @@
+create
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder add-dataProvider`
```

Comment:
`appbuilder add field` or `appbuilder add-field` - I'm not sure what is the correct syntax